packages/chips/README.md

```diff
@@ -0,0 +1,46 @@
+# React Chips
+
+A React version of an [MDC Chips](https://github.com/material-components/material-components-web/tree/master/packages/mdc-chips).
+
+## Installation
+
+```
+npm install @material/react-chips
+```
+
+## Usage
+
+```js
+import React from 'react';
+import ChipSet from '@material/react-chips';
+
+class MyApp extends React.Component {
+  render() {
+    return (
+      <ChipSet labels={['Chip One', 'Chip Two']} />
+    );
+  }
+}
+```
+
+## Props
+
+### Chip Set
+
+Prop Name | Type | Description
+--- | --- | ---
+className | String | Classes to be applied to the chip set element.
+labels | Array | Array of text to be displayed in each chip.
```

Comment:
Language in the description is a little ambiguous. Maybe "An array of strings. Each string has a corresponding chip whose label will be set to the value of that string."